packages/api/package.json

```diff
@@ -8,10 +8,11 @@
   "scripts": {
     "deploy": "wrangler publish --env production",
     "predev": "./scripts/cli.js services start && ./scripts/cli.js db-sql --cargo --testing --reset && ./scripts/cli.js minio bucket create dotstorage-dev-0",
-    "dev": "miniflare dist/worker.js --watch --debug --env ../../.env",
+    "dev": "miniflare dist/worker.js --watch --debug --env ../../.env --r2 CARPARK",
     "dev:persist": "PERSIST_VOLUMES=true npm run dev",
     "build": "scripts/cli.js build",
     "test": "./docker/run-with-dependencies.sh ./scripts/run-test.sh",
+    "posttest": "tsc",
```

Comment:
nit: we usually do this in pretest. I have no strong opinion, just raising for keeping things uniform